src/lib/swapper/swappers/OsmosisSwapper/OsmosisSwapper.ts

```diff
@@ -310,13 +306,29 @@ export class OsmosisSwapper implements Swapper<ChainId> {
       ? 'uosmo'
       : atomOnOsmosisAssetId.split('/')[1].replace(/:/g, '/')
 
+    const maybeRateInfo = await getRateInfo(
+      sellAsset.symbol,
+      buyAsset.symbol,
+      sellAmountCryptoBaseUnit !== '0' ? sellAmountCryptoBaseUnit : '1',
+      osmoUrl,
+    )
+
+    if (!maybeRateInfo.isOk()) return Err(maybeRateInfo.unwrapErr())
+    const { buyAmountCryptoBaseUnit } = maybeRateInfo.unwrap()
```

Comment:
Used for the IBC transfer in case of an OSMO -> ATOM swap (i.e the second Tx after the `swap-exact-amount-in` tx.
We know the sell amount, but we don't know the buy amount in this method, so we're using the rate to infer it.